### DISCUSSION TITLE
Notification Option For Developers (Admin)

### DISCUSSION THREAD

**Discussion by captain-hayze** [aws/aws-cli#7404] [Q&A] [answered]
### Describe the feature

A service of persistent notification and updating AWS admins of crucial and important activities potential outage.

### Use Case

1. Potential outages of essential services due to whatever reason.
2. Notification of 

### Proposed Solution

~whatsapp + phone calls

### Other Information

Configurability is crucial

### Acknowledgements

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CLI version used

aws-cli/2.4.23

### Environment details (OS name and version, etc.)

macOS Monterey, Version 12.4, Chip Apple M1

**Comment by kdaily** [comment, marked as answer]
Hi @captain-hayze,

Thanks for writing! Fortunately this does exist already:

https://aws.amazon.com/premiumsupport/technology/aws-health-dashboard/

The Service Health dashboard gives current and historical listing of any outages or issues with AWS.

There is an API already for this service ([link](https://docs.aws.amazon.com/health/latest/APIReference/Welcome.html)), but this does require a Business, Enterprise On-Ramp, or Enterprise Support plan from [AWS Support](http://aws.amazon.com/premiumsupport/).

Hope this answers your question! Since this is a guidance question, I'm going to move it to the discussion forum for others to find. Please feel free to mark it as answered or follow up with any other information. Thanks!
